### PR TITLE
feat(clearinghouse): add comments to process step and application checklist

### PR DESCRIPTION
**Pull request by Phil91**
## Description

* enhance application checklist with a comment of the skipped self description step
* adjust selfDescription retriever to check if clearinghouseConnect is disabled

## Why

To give the user an overview why the step was skipped

## Issue

Refs: #837

## Corresponding CD PR

https://github.com/eclipse-tractusx/portal/pull/400

## Checklist

- [x] I have followed the [contributing guidelines](https://github.com/eclipse-tractusx/portal-assets/blob/main/docs/developer/Technical%20Documentation/Dev%20Process/How%20to%20contribute.md#commit-and-pr-guidelines)
- [x] I have performed [IP checks](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-04#checking-libraries-using-the-eclipse-dash-license-tool) for added or updated 3rd party libraries
- [x] I have created and linked IP issues or requested their creation by a committer
- [x] I have performed a self-review of my own code
- [x] I have successfully tested my changes locally
- [x] I have added tests that prove my changes work
- [x] I have checked that new and existing tests pass locally with my changes
- [x] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

**Review by evegufy — COMMENTED** [review]
Please change to main branch, milestone was already adjusted

**Comment by Phil91** [comment]
> Please change to main branch, milestone was already adjusted

rebase to main done.

**Review by evegufy — APPROVED** [review]

**Comment by evegufy** [comment]
@Phil91 I noticed that a conflict needs to get resolved

**Comment by Phil91** [comment]
> @Phil91 I noticed that a conflict needs to get resolved

@evegufy conflict is resolved :)

File: src/administration/Administration.Service/appsettings.json [hunk 1]
```diff
@@ -257,7 +257,8 @@
   "Registration": {
     "ApplicationsMaxPageSize": 20,
     "DocumentTypeIds": [],
-    "UseDimWallet": false
+    "UseDimWallet": false,
+    "ClearinghouseConnectDisabled": false
```
Comment (evegufy) [inline src/administration/Administration.Service/appsettings.json:261]: as you added the value in the administration service here, does it make sense to also add it to the processes worker?: https://github.com/eclipse-tractusx/portal-backend/blob/v2.1.0-RC2/src/processes/Processes.Worker/appsettings.json#L326
issue https://github.com/eclipse-tractusx/portal-backend/issues/888

**Review by evegufy — CHANGES_REQUESTED** [review]

File: src/administration/Administration.Service/appsettings.json (hunk 1 above)
Comment (Phil91) [reply src/administration/Administration.Service/appsettings.json:261]: added

Comment (evegufy) [reply src/administration/Administration.Service/appsettings.json:263]: @Phil91 could you please also add it under ApplicationChecklist__SdFactory?
https://github.com/eclipse-tractusx/portal/blob/b5caed1cc33aefadc33c9c634b6b700f239be35b/charts/portal/templates/deployment-backend-administration.yaml#L179

**Review by ntruchsess — APPROVED** [review]

File: src/administration/Administration.Service/appsettings.json (hunk 1 above)
Comment (Phil91) [reply src/administration/Administration.Service/appsettings.json:263]: I've added it: https://github.com/eclipse-tractusx/portal-backend/blob/51834cb2977be614b125748bfaffa589f971420c/src/administration/Administration.Service/appsettings.json#L321 or am I missing something?

**Review by evegufy — APPROVED** [review]

**Pull request merged**